ui/app/src/utils/xml.ts

```diff
@@ -183,14 +183,17 @@ export function deserialize(xml: Node): any;
 export function deserialize(xml: string, options: X2jOptionsOptional): any;
 export function deserialize(xml: Node, options: X2jOptionsOptional): any;
 export function deserialize(xml: string | Node, options?: X2jOptionsOptional): any {
-  if (typeof xml !== 'string') {
-    xml = serialize(xml);
-  }
-  return parser.parse(xml, {
+  const parser = new XMLParser({
     attributeNamePrefix: '',
     ignoreAttributes: false,
+    htmlEntities: true,
```

Comment:
Please test what happens if there are html entities inside CDATA, also test if the ui.xml UM/CDATA issues get fixed by this